spec/support/matchers/declaration.rb

```diff
@@ -48,7 +48,7 @@ def with_options(options)
     def failure_message
       [
         "expected declarations to include declaration of type #{@declaration_type}",
-        @options ? "with options #{options}" : nil
+        @options ? "with options #{options}" : nil,
```

Comment:
Perhaps out of scope for this PR, but I'd prefer if this ternary were replaced with 
```ruby
@options && "with options #{options}"
```

Reply:
I agree with you, but I do think it's out of the scope of the PR. Trying to keep it focused to only fixing Rubocop offenses.